src/types/mongoose/settings.ts

```diff
@@ -0,0 +1,34 @@
+import { Model, HydratedDocument } from 'mongoose';
+
+export interface ISettings {
+    pid: number;
```

Comment:
Putting this here but it applies everywhere, we use tabs of size 4

Reply:
I thought I had this set in eslint config, I'll fix it across all the files

Reply:
Fixed this, looks like eslint doesn't check `.d.ts` files so these were never automatically caught / formatted. Just manually fixed them all in a new commit